pyproject.toml

```diff
@@ -89,9 +89,9 @@ gql = {version = "^3.4.1", optional = true}
 pandas = {version = "^2.0.1", optional = true}
 telethon = {version = "^1.28.5", optional = true}
 zep-python = {version="^0.25", optional=true}
+langkit = {version = "^0.0.1.dev3", optional = true}
```

Comment:
this needs to be listed in the one of the extra installs (probably `all`)

Reply:
Done: Added langkit to `all` group, updated PR branch from main repo and rebased to help lower conflicts, reran `poetry lock --no-update`, reran through tests and notebook example.